backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc

```diff
@@ -46,9 +46,51 @@ The CephFS shallow copy feature is a back up feature. It is not part of restore
 ====
 
 include::modules/oadp-ceph-prerequisites.adoc[leveloffset=+2]
-include::modules/oadp-ceph-preparing-crs.adoc[leveloffset=+2]
-include::modules/oadp-ceph-cephfs.adoc[leveloffset=+2]
-include::modules/oadp-ceph-split.adoc[leveloffset=+2]
+
+[id="defining-crs-for-12-data-mover"]
+=== Defining custom resources for use with OADP 1.2 Data Mover
+
+When you install {rh-storage-first}, it automatically creates default CephFS and a CephRBD `StorageClass` and `VolumeSnapshotClass` custom resources (CRs). You must define these CRs for use with OpenShift API for Data Protection (OADP) 1.2 Data Mover.
+
+After you define the CRs, you must make several other changes to your environment before you can perform your back up and restore operations.
+
+include::modules/oadp-ceph-preparing-cephfs-crs.adoc[leveloffset=+2]
+include::modules/oadp-ceph-preparing-cephrbd-crs.adoc[leveloffset=+2]
+include::modules/oadp-ceph-preparing-crs-additional.adoc[leveloffset=+2]
+
+[id="oadp-ceph-back-up-restore-cephfs"]
+=== Backing up and restoring data using OADP 1.2 Data Mover and CephFS storage
+
+You can use OpenShift API for Data Protection (OADP) 1.2 Data Mover to back up and restore data using CephFS storage by enabling the shallow copy feature of CephFS.
+
+include::snippets/oadp-ceph-cr-prerequisites.adoc[]
+
+:context: !backing-up-applications
+
+:context: cephfs
+
+include::modules/oadp-ceph-cephfs-back-up-dba.adoc[leveloffset=+2]
+include::modules/oadp-ceph-cephfs-back-up.adoc[leveloffset=+2]
+include::modules/oadp-ceph-cephfs-restore.adoc[leveloffset=+2]
+
+[id="oadp-ceph-split"]
+=== Backing up and restoring data using OADP 1.2 Data Mover and split volumes (CephFS and Ceph RBD)
+
+You can use OpenShift API for Data Protection (OADP) 1.2 Data Mover to back up and restore data in an environment that has _split volumes_, that is, an environment that uses both CephFS and CephRBD.
```

Comment:
It might be worth creating a variable for the OADP 1.2 Data Mover. Just something to consider to make maintenance easier in the future.

Reply:
Hopefully, we'll soon start referring only to the "OADP Data Mover," at which point an attribute would be a good idea. Not that it wouldn't be a good idea now :)